File: Lições/pascal/EXERCICIO02.pas
Program EXERCICIO02;

// 02) Escreva um algoritmo que receba as medidas do comprimento e da largura de um terreno.
// Calcule e exiba a área do terreno.

Var
comprimento, largura, area: real;

Begin
  WRITE('Digite o comprimento do terreno: ');
  READ(comprimento);
  WRITE('Digite a largura do terreno: ');
  READ(largura);
  area := comprimento * largura ;
  WRITE('A área do seu terreno é igual a ',area:2:2,'m²');
End.

File: Lições/c/EXERCICIO02.c
// 02) Escreva um algoritmo que receba as medidas do comprimento e da largura de um terreno. 
// Calcule e exiba a área do terreno.
#include <stdio.h>
#include <locale.h>
float comprimento, largura;

int main(){
	setlocale(LC_ALL,"portuguese");
	
	printf("Digite o valor do comprimento: ");
	scanf("%f",&comprimento);
	printf("Digite o valor da largura: ");
	scanf("%f",&largura);
	printf("A área do terreno é: %.2f m²",comprimento * largura);
	
	return 0;
}
